data-prepper-plugins/service-map-stateful/README.md

```diff
@@ -1,13 +1,27 @@
 # Service-Map Stateful Prepper
 
-This is a special prepper that consumes Opentelemetry traces, stores them in a LMDB data store and evaluate relationships at fixed ```window_duration```. 
+This is a special prepper that consumes Opentelemetry traces, stores them in a MapDB data store and evaluate relationships at fixed ```window_duration```. 
 The lmdb databases are stored in the ```data/service-map/*``` path.
+
+# Usages
+Example `.yaml` configuration:
 ```
 prepper:
     service-map-stateful:
 ```
 
 ## Configurations
 
-* window_duration => In seconds. Default is ```180```. 
+* window_duration(Optional) => An `int` represents the fixed time window in seconds to evaluate service-map relationships. Default is ```180```. 
+
+## Metrics
+Besides common metrics in [AbstractPrepper](https://github.com/opendistro-for-elasticsearch/data-prepper/blob/master/data-prepper-api/src/main/java/com/amazon/dataprepper/model/prepper/AbstractPrepper.java), peer-forwarder introduces the following custom metrics.
```

Comment:
I believe peer-forwarder is a typo, it should be service-map-stateful introduces the following custom metrics.